examples/image_streamer/golden_image.rb

```diff
@@ -0,0 +1,84 @@
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+oneview_client = {
+  url: ENV['ONEVIEWSDK_URL'],
+  user: ENV['ONEVIEWSDK_USER'],
+  password: ENV['ONEVIEWSDK_PASSWORD']
+}
+
+i3s_client = {
+  url: ENV['I3S_URL'],
+  oneview_client: oneview_client
+}
+
+# Create or update the Golden Image 'GoldenImage1'
+image_streamer_golden_image 'GoldenImage1' do
+  client i3s_client
+  api_version 300
```

Comment:
Would it be easier to just set `node.default['oneview']['api_version'] = 200` at the top of all the I3S recipes instead of adding this attribute to each example?